core/src/main/java/io/grpc/transport/AbstractClientStream.java

```diff
@@ -125,8 +130,8 @@ protected void inboundHeadersReceived(Metadata.Headers headers) {
    *
    * @param frame the received data frame. Its ownership is transferred to this method.
    */
-  protected void inboundDataReceived(ReadableBuffer frame) {
-    Preconditions.checkNotNull(frame, "frame");
```

Comment:
@ejona86:  Java style guide says:
> You are advised not to comment on the following issues in code reviews:
> ...
> Import order. Don't itemize; just use the tools (see Section 9.3, Style-related tools).
> ...
> It is reasonable to ask the reviewee to check their editor configuration, but don't point out specific violations.

Code reviews are more of a sanity check, not an exact correctness check.  There is a list of [best practices](https://docs.google.com/document/d/1pG-WwM0cIWhH7SSuDXrAJ2JWX7JCYLI5HSRxMoTQPiY/), which includes

> Do not force your personal preferences. If you are suggesting to change something, there should be a clear win. If code complies with style guidelines and is not wrong, then code author is right.

It also stipulates that consistency in the code base should be enforced:
> Agree as a team to follow specific style where Java Style guide leaves it open-ended and enforce it

So, this means that the entire file should be changed at once, rather than just in single lines that changed.  It either needs to be up to the PR author, or consistently one way across the code, but not neither, and not both.   If you don't like the code review noise, then let's pick a hard-and-fast rule and follow it.